lib/room.cpp

```diff
@@ -52,18 +52,25 @@
 #include "converters.h"
 #include "syncdata.h"
 
+#include "e2ee.h"
+
 #include <QtCore/QHash>
 #include <QtCore/QStringBuilder> // for efficient string concats (operator%)
 #include <QtCore/QPointer>
 #include <QtCore/QDir>
 #include <QtCore/QTemporaryFile>
 #include <QtCore/QRegularExpression>
 #include <QtCore/QMimeDatabase>
+#include <QtCore/QJsonDocument>
```

Comment:
Sorry but I couldn't find where you use `QJsonDocument` below. Can you point me to the line?